n2y/blocks.py

```diff
@@ -67,6 +67,12 @@ def children_to_pandoc(self):
         for block_type, blocks in groupby(self.children, lambda c: type(c)):
             if issubclass(block_type, ListItemBlock):
                 pandoc_ast.append(block_type.list_to_pandoc(blocks))
+            elif block_type == ChildPageBlock:
+                for b in blocks:
+                    result = b.to_pandoc()
+                    title = result[0][0]['title'][0]
```

Comment:
Nice idea! I think it makes a lot of sense to expand out sub pages. I would actually not show the page title as an H1, and would just show the page's content in place. This would work identically to linked pages then, which we use (or have a plugin) that expands.

Reply:
yea, I only did it as a debug because [this](https://www.notion.so/Type-is-used-to-categorize-requirements-Possible-types-include-cde96c39f96141a9b43f165d422addbf) subpage was converting to pandoc as a Pandoc block inside of another Pandoc block and that's against protocol. Is there another way you'd solve that?

Reply:
I think this was due to a weird thing Emily did in the templates. I just fixed it in the templates! That said, I think expanding the page (without the title) is a good idea.